pkg/controllers/multiclusterservice/endpointslice_collect_controller.go

```diff
@@ -396,6 +391,40 @@ func reportEndpointSlice(c client.Client, endpointSlice *unstructured.Unstructur
 	return nil
 }
 
+func getEndpointSliceWorkMeta(c client.Client, ns string, workName string, endpointSlice *unstructured.Unstructured) (metav1.ObjectMeta, error) {
```

Comment:
Functions with the same name in pkg/controllers/mcs/service_export_controller.go have a high degree of repetition and can be extracted as common functions.

Reply:
There are differences in multiple segments of logic, making it difficult to extract a function.